tree/ntuple/v7/test/ntuple_storage_s3.cxx

```diff
@@ -0,0 +1,29 @@
+#include "ntuple_test.hxx"
+
+TEST(RNTuple, S3Basics)
+{
+   std::string s3Uri("s3://ntpl0.s3.us-east-2.amazonaws.com");
```

Comment:
It seems that the test failing in CI because we are not setting the `S3_xxx` environment variables?